lib/ui/views/home/home_viewmodel.dart

```diff
@@ -115,15 +115,14 @@ class HomeViewModel extends BaseViewModel {
   }
 
   Future<bool> hasManagerUpdates() async {
-    final String? latestVersion = await _managerAPI.getLatestManagerVersion();
     String currentVersion = await _managerAPI.getCurrentManagerVersion();
 
     // add v to current version
     if (!currentVersion.startsWith('v')) {
       currentVersion = 'v$currentVersion';
     }
 
-    if (latestVersion != currentVersion) {
+    if (_latestManagerVersion != currentVersion) {
```

Comment:
You are now comparing the current version of the manager (see comment above) with the current version of the manager.